jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/TabsTests.java

```diff
@@ -75,12 +77,12 @@ public static void centerActiveTabsTest() {
 		jdiAssert(centerActiveTabs.navigation().nextButtonIsActive(), Matchers.is(true),
 				"Next button is disabled");
 		centerActiveTabs.get(1).is().visible();
-		centerActiveTabs.get(13).is().notVisible();
+		centerActiveTabs.get(17).is().notVisible();
```

Comment:
Why this number is better then 13?

Reply:
A magic one ;)

17 corresponds nicely  to the window size 1920x1080 set above in the before() method.

In this case tab 17 will be invisible indeed when the first tab is selected. Which didn't happen to tab 13 as it remained visible for that size of the window. This was the cause of the test failure.

The test was initially done and tested for a smaller screen, I guess for Mac 13', this is why it passed locally.

Because on the CI now 1920x1080 browser window size is used I thought  it would be good to explicitly set the same size for this test. This would prevent the test from failure locally and on the CI independently on window size value set in the configuration file.